Sources/TensorFlow/Layer.swift

```diff
@@ -191,3 +191,35 @@ public final class Parameter<Scalar: TensorFlowScalar> {
         self.value = value
     }
 }
+
+/// A wrapper around a differentiable value with toggleable trainability.
+@_propertyWrapper
+public struct Trainable<Value: Differentiable> : Differentiable {
+    @noDerivative public var isTrainable: Bool = true
+    public var _value: Value
+
+    public init(initialValue: Value) {
+        _value = initialValue
+    }
+
+    @differentiable(vjp: _vjpValue)
+    public var value: Value {
+        get { _value.withoutDerivative() }
```

Comment:
Hmm, I don't see the point of calling `withoutDerivative()` while a VJP is provided.

Reply:
Oops, removed.